### PR TITLE
:art: Add comments in Layouts.tsx, Layout.ts, Root.tsx, classic.blocks/_index.tsx, Unit.ts, Config.tsx, Format.tsx

### PR DESCRIPTION
**Pull request by crapper**

### CODE REVIEW

File: src/format/Format.tsx [hunk 1]
```diff
@@ -21,16 +21,45 @@ export interface Format {
...
+  /**
+   * A initialize function triggered when MainApp changes format
+   * This will register/records all possible disposer to Command listeners and UI rendered for this format
+   * @param app The MainApp component
+   * @param ui The UserInterface component
+   */
   register(app: MainApp, ui: UserInterface): void;
 
+  /**
+   * A dispose function triggered when MainApp changes format
```
Comment (Jerrylum) [inline src/format/Format.tsx:33]: Use the following
```
Unregisters the format from the application.

This should clean up and remove any hooks, event listeners, or UI components that were added during the registration process.

The `unregister` function should be the last function to call on the format. After it is called, the format is expected to be detached from the application. This is triggered when MainApp changes format.
```

File: src/format/Format.tsx [hunk 2]
```diff
@@ -196,7 +225,7 @@ const convertFromV0_7_0ToV0_8_0: PathFileDataConverter = {
 const convertFromV0_8_0ToCurrentAppVersion: PathFileDataConverter = {
   version: new Range("~0.8"),
   convert: (data: Record<string, any>): void => {
-    // From v0.7.0 to current app version
+    // From v0.8.0 to current app version
```
Comment (Jerrylum) [inline src/format/Format.tsx:228]: Nice one

File: src/app/classic.blocks/_index.tsx
Comment (Jerrylum) [inline src/app/classic.blocks/_index.tsx:1]: I think you can remove this comment

**Review by Jerrylum — CHANGES_REQUESTED** [review]
Everything is good, including the PR title, please address the following feedbacks

File: src/app/classic.blocks/_index.tsx
Comment (crapper) [reply src/app/classic.blocks/_index.tsx:1]: Removed.

File: src/format/Format.tsx (hunk 1 above)
Comment (crapper) [reply src/format/Format.tsx]: Updated, thank you!

File: src/format/Format.tsx [hunk 3]
```diff
@@ -21,16 +21,45 @@ export interface Format {
 
   getDescription(): string;
 
+  /**
+   * A initialize function triggered when MainApp changes format
```
[the remark this answers is not in the document]
Comment (crapper) [reply src/format/Format.tsx]: Updated, thank you!

File: src/format/Format.tsx [hunk 4]
```diff
@@ -21,16 +21,45 @@ export interface Format {
...
   register(app: MainApp, ui: UserInterface): void;
 
+  /**
+   * A dispose function triggered when MainApp changes format
+   * This will call all the disposer to Command listeners and UI rendered for this format
+   */
   unregister(): void;
 
+  /**
+   * Create a new instance of this format
```
[the remark this answers is not in the document]
Comment (crapper) [reply src/format/Format.tsx]: Updated, thank you!

File: src/format/Format.tsx [hunk 5]
```diff
@@ -21,16 +21,45 @@ export interface Format {
...
   createNewInstance(): Format;
 
+  /**
+   * Get the general configuration for this format
+   * @returns the general configuration for this format
+   */
   getGeneralConfig(): GeneralConfig;
 
+  /**
+   * Create a path instance with the given segments
```
[the remark this answers is not in the document]
Comment (crapper) [reply src/format/Format.tsx]: Updated, thank you!

File: src/format/Format.tsx [hunk 6]
```diff
@@ -21,16 +21,45 @@ export interface Format {
...
+   * @param segments the segments to create the path
+   * @returns the created path instance
+   */
   createPath(...segments: Segment[]): Path;
 
+  /**
+   * Get the points along a path at a uniform density
```
[the remark this answers is not in the document]
Comment (crapper) [reply src/format/Format.tsx]: Updated, thank you!

File: src/format/Format.tsx [hunk 7]
```diff
@@ -21,16 +21,45 @@ export interface Format {
 
   getDescription(): string;
 
+  /**
+   * A initialize function triggered when MainApp changes format
+   * This will register/records all possible disposer to Command listeners and UI rendered for this format
+   * @param app The MainApp component
```
[the remark this answers is not in the document]
Comment (crapper) [reply src/format/Format.tsx]: Double-checked; I believe here is the only place I misused the term instance vs. component. Thank you very much!!!

**Pull request merged**